cmd/model_downloader/model_resolver.go

```diff
@@ -13,15 +13,29 @@ func main() {
 		"model URL, path, or huggingface id to fetch")
 	destPath := flag.String("dest", "./",
 		"where to download the model to")
+	modelType := flag.String("type", "transformers",
+		"model type (transformers or diffusers)")
 	flag.Parse()
 	if *modelId == "" {
 		flag.Usage()
 		log.Fatal("Must provide -model")
 	}
 
+	// map modelType to resource type enum
+	var rsrcType resources.ResourceType
+	switch *modelType {
+	case "transformers":
+		rsrcType = resources.RESOURCETYPE_TRANSFORMERS
+	case "diffusers":
+		rsrcType = resources.RESOURCETYPE_DIFFUSERS
+	default:
+		flag.Usage()
+		log.Fatal("Invalid -type")
```

Comment:
Maybe do a `Fatalf` or `Fatal(fmt.sprinter for the invalid argument passed to make it a little more friendly.